scripts/voice/deepstack_brain.py

```
@@ -0,0 +1,279 @@
+- If data is missing or unavailable, say so honestly.
+- Think in risk/reward. Always consider downside.
+- Use trader vernacular but remain clear.
+- When citing numbers, always specify units (cents, dollars, percentage).
+- For Telegram HTML: convert **bold** to <b>bold</b>, `code` to <code>code</code>.
+  Output as markdown — the system handles conversion.
+"""
+
+
+def ask_brain(
+    message: str,
+    intent_type: str = "general_chat",
```

Ruff (0.14.14)

[warning] Unused function argument: `intent_type`

(ARG001)